### DISCUSSION TITLE
User postgres 16?

### DISCUSSION THREAD

**Discussion by theodufort** [supabase/supabase#30535] [Questions]
Does anyone know how to use PostgreSQL 16? 
I looked on the Docker Hub and no image is available except under PG15.

**Comment by GaryAustin1** [comment]
Supabase is not supporting yet.
But appears work is going on...
https://github.com/supabase/postgres/pull/1279
If you search pull requests you can see they have been working on blockers.

**Reply by theodufort** [reply]
Ok so I guess ill wait patiently ahah
